apps/spruce/src/pages/version/TaskDuration.tsx

```diff
@@ -101,12 +100,18 @@ const TaskDuration: React.FC<Props> = ({ taskCount }) => {
         page={page}
         totalCount={taskCount}
       />
-      <TaskDurationTable
-        loading={loading}
-        // @ts-expect-error: FIXME. This comment was added by an automated script.
-        numLoadingRows={limit}
-        tasks={tasksData}
-      />
+      {
+        // Ensures that the TaskDurationTable initial sort
+        // button states intialize with the correct default values.
+        hasInitialized && (
```

Comment:
Would it be possible to check if there is a sort defined instead of adding this new variable?

Reply:
It's not possible. The `sorts` query param may be defined in the URL from the task table so it's not sufficient to check whether the variable is defined. This worked without this before because the code was setting the default value twice (once in this useEffect and once in getInitialParams) and also because the TaskDuration and Tasks table utilized different query params for the initial value (sortBy/sortDir vs sorts)